.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Triggering CI from every commit and PR would be better I guess:

```suggestion
  - push
  - pull_request
```